astromon/telemetry.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
```

Comment:
Should this entire module even be here? It feels out of place, like something that is specific to the periscope correlation analysis only. The functions here are not used anywhere else in the package from what I could see.